staging/src/k8s.io/api/storage/v1beta1/types.go

```diff
@@ -349,7 +353,17 @@ type CSINodeDriver struct {
 	// It is possible for different nodes to use different topology keys.
 	// This can be empty if driver does not support topology.
 	// +optional
-	TopologyKeys []string `json:"topologyKeys" protobuf:"bytes,3,rep,name=topologyKeys"`
+	TopologyKeys []string `json:"topologyKeys" protobuf:"bytes,4,rep,name=topologyKeys"`
```

Comment:
I don't think you can change protobuf field numbers like this, that's going to break existing serialized objects. (use 4 for allocatable)